test/cpp/end2end/end2end_test.cc

```diff
@@ -337,7 +338,11 @@ class End2endTest : public ::testing::TestWithParam<TestScenario> {
     }
   }
 
-  void ResetStub() {
+  bool ResetStub() {
+    if (GetParam().callback_server && !GetParam().inproc &&
```

Comment:
Any reason this can't be done in SetUp like the callback test suite ? When will this change in the middle of test run